test/dynamo/test_repros.py

```diff
@@ -2226,6 +2227,27 @@ def f(x):
         )
         self.assertEqual(gm(inp).shape, f(inp).shape)
 
+    def test_cond_with_aten_mode(self):
+        def pred(x):
+            return x.all()
+
+        def true_fn(x):
+            return x * x
+
+        def false_fn(x):
+            return x + x
+
+        def my_fn(x):
+            return cond(x.all(), true_fn, false_fn, [x])
```

Comment:
PR changes cond to accept tuple. This test is testing a list of Tensors. Do we want to test a tuple of Tensors?

Reply:
I plan to unify this behaviour in the next diff.

Reply:
Cool, thanks for the confirmation that we are testing it. It would be good to write a separate unittest to check that in eager mode, cond can accept a tuple of Tensors.

Also, orthogonal, but -- The dynamo behaviour sounds wrong to me. It shouldn't be rewriting the args to a function to have a different type.

Reply:
Yep there are some inconsistencies. I plan to address all these container related issues for once an d all in the following diff.